classes/book.lua

```diff
@@ -63,6 +65,7 @@ end
 
 function book:endPage ()
   self:moveTocNodes()
+  self:moveLabelRefs()
```

Comment:
Will the `endpage` hook location I added in #1414 work for this?

Reply:
Done.